gnssanalysis/gn_io/sp3.py

```diff
@@ -35,6 +32,31 @@
 # File descriptor and clock
 _RE_SP3_HEAD_FDESCR = _re.compile(rb"\%c[ ]+(\w{1})[ ]+cc[ ](\w{3})")
 
+
+_SP3_DEF_PV_WIDTH = [1, 3, 14, 14, 14, 14, 1, 2, 1, 2, 1, 2, 1, 3, 1, 1, 1, 1, 1, 1]
```

Comment:
Good stuff. Thanks for this, I'd been meaning to make the use of FWF more robust.